entrypoint.sh

```diff
@@ -1,6 +1,13 @@
 #!/bin/bash
 
-TFSEC_OUTPUT=$(/go/bin/tfsec /github/workspace)
+# Set Terraform working directory
+if [ "${INPUT_TFSEC_ACTIONS_WORKING_DIR}" != "" ] || [ "${INPUT_TFSEC_ACTIONS_WORKING_DIR}" != "." ]; then
```

Comment:
This if is always returning `True`, I think you might want to replace `||` by `&&`

Reply:
Thanks for the feedback @triat, that was a silly mistake on my part :man_facepalming: 
This should be corrected now

Reply:
Don't worry, this is exactly why there is PR 👍 

Thanks for the quick update